pkgs/top-level/all-packages.nix

```diff
@@ -29242,7 +29242,9 @@ with pkgs;
 
   qemacs = callPackage ../applications/editors/qemacs { };
 
-  roxctl = callPackage ../applications/networking/cluster/roxctl { };
+  roxctl = callPackage ../applications/networking/cluster/roxctl {
+    buildGoModule = buildGo117Module;
```

Comment:
Why was this done?

Reply:
It builds and runs with current go on darwin/linux so I'm removing it in: https://github.com/NixOS/nixpkgs/pull/188445

Reply:
`1.17` is still the version of the upstream release (see https://github.com/stackrox/stackrox/blob/master/EXPECTED_GO_VERSION). But should be fine to unpin if it builds.